cunningham.ts

```diff
@@ -308,6 +308,36 @@ const config = {
           "value-color": "ref(theme.colors.greyscale-950)",
           "font-size": "14px",
         },
+        "badge": {
+          "font-size": "var(--c--theme--font--sizes--xs)",
```

Comment:
we should use `ref` instead of `var`. Otherwise when generating tokens in TS for instance it would not give the accurate value, but just `var(...)` instead

Reply:
Oh right! There are several other components where we are using `var(...)`. Do you I want fix all in one ?

Reply:
why not but not mandatory